RecoVertex/Configuration/python/RecoVertex_cff.py

```diff
@@ -23,11 +23,52 @@
+unsortedOfflinePrimaryVertices1D.TkFilterParameters.minPt = cms.double(1.0)
+offlinePrimaryVertices1D=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices1D", particles="trackRefsForJetsBeforeSorting")
+offlinePrimaryVertices1DWithBS=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices1D:WithBS", particles="trackRefsForJetsBeforeSorting")
+DA2DParameters.TkDAClusParameters.verbose = cms.untracked.bool(False)
+unsortedOfflinePrimaryVertices4D = unsortedOfflinePrimaryVertices.clone( verbose = cms.untracked.bool(False),
+                                                                         TkClusParameters = DA2DParameters )
+unsortedOfflinePrimaryVertices4D.TkFilterParameters.minPt = cms.double(1.0)
+unsortedOfflinePrimaryVertices4D.TrackTimesLabel = cms.InputTag("trackTimeValueMapProducer:generalTracksConfigurableFlatResolutionModel")
+unsortedOfflinePrimaryVertices4D.TrackTimeResosLabel = cms.InputTag("trackTimeValueMapProducer:generalTracksConfigurableFlatResolutionModelResolution")
+offlinePrimaryVertices4D=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4D", particles="trackRefsForJetsBeforeSorting")
+offlinePrimaryVertices4DWithBS=sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices4D:WithBS", particles="trackRefsForJetsBeforeSorting")
+
+from SimTracker.TrackerHitAssociation.tpClusterProducer_cfi import *
+from SimTracker.TrackAssociatorProducers.quickTrackAssociatorByHits_cfi import *
+from SimTracker.TrackAssociation.trackTimeValueMapProducer_cfi import *
+from Configuration.StandardSequences.Eras import eras
+_phase2_tktiming_vertexreco = cms.Sequence( vertexreco.copy() *
+                                            tpClusterProducer *
+                                            quickTrackAssociatorByHits *
+                                            trackTimeValueMapProducer *
+                                            unsortedOfflinePrimaryVertices1D *
+                                            offlinePrimaryVertices1D *
+                                            offlinePrimaryVertices1DWithBS *
+                                            unsortedOfflinePrimaryVertices4D *
+                                            offlinePrimaryVertices4D *
+                                            offlinePrimaryVertices4DWithBS 
+                                            )
+
+eras.phase2_tracker.toModify( quickTrackAssociatorByHits,
+                              pixelSimLinkSrc = cms.InputTag("simSiPixelDigis","Pixel"),
+                              stripSimLinkSrc = cms.InputTag("simSiPixelDigis","Tracker")
+                              )
+
+eras.phase2_tracker.toModify( tpClusterProducer,
+                              pixelSimLinkSrc = cms.InputTag("simSiPixelDigis", "Pixel"),
+                              phase2OTSimLinkSrc = cms.InputTag("simSiPixelDigis","Tracker")
+                              )
```

Comment:
quickTrackAssociatorByHits and tpClusterProducer modifications do not belong in here.
Move to quickTrackAssociatorByHits_cfi and trackTimeValueMapProducer_cfi